index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+   <meta charset="utf-8">
+   <meta name="viewport" content= "width=device-width, initial-scale=1.0">
+   <title>Norman Borlaug</title>
+   <link rel="stylesheet" href="style.css">
+ </head>
+ <body>
+   <div class="main">
+     <div class="head">
+       <h1>Dr. Norman Borlaug</h1>
+       <p>The man who saved a billion lives</p>
+     </div>
+   <figure class = "images">
+       <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists">
+       <figcaption>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+   </figure>
```

Comment:
⋅⋅* Add alt text for the image

Reply:
- [ ] @Abdullah2213565 good review but I would suggest the use of polite words like please kindly when requesting a change, and also explain why the change needs to be done.